pkg/oauth/external/github/github_test.go

```diff
@@ -0,0 +1,186 @@
+package github
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/RangelReale/osincli"
+	"github.com/openshift/oauth-server/pkg/api"
+)
+
+func TestGetUserIdentity(t *testing.T) {
```

Comment:
Nit: In general I like your code very much, but I don't like to put everything into the TestFunc. I would hope that a lot of the code can be reused later on :smile:

Reply:
Nothing prevents the future developer to idiomatically copy-paste! Or worst case, to extract into a function and reuse :P